app/Policies/AchievementAuthorPolicy.php

```diff
@@ -40,7 +42,9 @@ public function create(User $user): bool
 
     public function update(User $user, AchievementAuthor $achievementAuthor): bool
     {
-        return $this->manage($user);
+        return
+            $this->manage($user)
```

Comment:
Is this `manage` check still desirable?

Reply:
Good catch - it is now redundant. Removed in latest.